irohad/network/impl/block_loader_impl.cpp

```diff
@@ -39,33 +39,16 @@ namespace {
 
 BlockLoaderImpl::BlockLoaderImpl(
     std::shared_ptr<PeerQueryFactory> peer_query_factory,
-    std::shared_ptr<BlockQueryFactory> block_query_factory,
     shared_model::proto::ProtoBlockFactory factory)
     : peer_query_factory_(std::move(peer_query_factory)),
-      block_query_factory_(std::move(block_query_factory)),
       block_factory_(std::move(factory)),
       log_(logger::log("BlockLoaderImpl")) {}
 
 rxcpp::observable<std::shared_ptr<Block>> BlockLoaderImpl::retrieveBlocks(
+    const shared_model::interface::types::HeightType &height,
```

Comment:
No [need](https://stackoverflow.com/questions/4143644/c-64-bit-int-pass-by-reference-or-pass-by-value) to pass `uint_64_t`, which is `HeightType`, by reference